lib/metadata/schema-parser.js

```diff
@@ -732,15 +766,69 @@ SchemaParserV2.prototype._parseKeyspace = function (row) {
     row['keyspace_name'],
     row['durable_writes'],
     strategy,
-    strategyOptions);
+    strategyOptions,
+    virtual);
 };
 
 /** @override */
-SchemaParserV2.prototype._parseTableOrView = function (tableInfo, tableRow, columnRows, indexRows, callback) {
+SchemaParserV2.prototype._parseTableOrView = function (tableInfo, tableRow, columnRows, indexRows, virtual, callback) {
```

Comment:
Could have implemented a `SchemaParserV3.prototype._parseTableOrView` override, but figured it would be fine to do here as older C* versions would never pass `virtual = true` anyways.

Reply:
Good call 👍